index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>My Portfolio</title>
+</head>
+
+<body>
+    <div>
+        <header class="header flex-row">
+            <div class="menu-button">
+                <a class="logo">JEPM</a>
+            </div>
+            <nav class="menu-mobile">
+                <button type="button" class="menu-button"><img src="./img/header-menu.png" alt="menu-image"></button>
+            </nav>
+        </header>
+        <section class="headline main">
+            <h1 class="presentation"> I'm Jorge E. Prada M.<br>
+                Glad to see you!<br>
+            </h1>
```

Comment:
- [ ] Good job with the correct semantics tag used on your hero text. However, I think it would be better if you update its styling by using the correct `color`, `fonts(size, weight)`, `line height`, `spacing around`(padding & margin),... Just ensure that your styling matches the one from Figma template.

Reply:
Enter the requested characteristics